tests/agent_server/test_server_agent.py

```diff
@@ -1889,24 +1889,6 @@ async def test_stop_autostarted_agents_on_environment_removal(server, client, re
     ps_diff_inmanta_agent_processes(original=inmanta_agent_child_processes, current_process=current_process, diff=0)
 
 
-async def test_stop_autostarted_agents_on_project_removal(server, client, resource_container, no_agent_backoff):
```

Comment:
Why was this test case removed?

Reply:
I see. We have another test case for the environment. You can ignore this comment.